modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -10,9 +10,14 @@ A cluster administrator can configure https://www.haproxy.org/download/1.8/doc/p
 
 This feature is not supported in cloud deployments. This restriction is because when {product-title} runs in a cloud platform, and an IngressController specifies that a service load balancer should be used, the Ingress Operator configures the load balancer service and enables the PROXY protocol based on the platform requirement for preserving source addresses.
 
+[IMPORTANT]
+====
+You must configure both {product-title} and the external load balancer to either use the PROXY protocol, or to use TCP.
+====
+
 [WARNING]
 ====
-To prevent connection failure, configure both the Ingress Controller and the load balancer to use the PROXY protocol.
+The PROXY protocol is unsupported for the default Ingress Controller with IPI clusters on non-cloud platforms that use a Keepalived Ingress VIP.
```

Comment:
@kalexand-rh @jldohmann 

I think we're avoiding the terms IPI and UPI? Isn't this still _installer-provisioned clusters_ here?

Thanks.